test/sagemaker_tests/pytorch/training/resources/mnist/smdebug_mnist.py

```diff
@@ -33,6 +33,7 @@
 from smdebug.pytorch import *
 import numpy as np
 import random
+from packaging import version
```

Comment:
nit: Please use `from packaging.version import Version`, as the `Version` class is more intuitive in comparing version strings than `version.parse`.